hack/deploy.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
+	OPERATOR_IMAGE="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:cincinnati-operator"
+	GRAPH_DATA_IMAGE="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:cincinnati-graph-data-container"
+
+	echo "Openshift CI detected, deploying using image $OPERATOR_IMAGE and ${GRAPH_DATA_IMAGE}"
+
+	sed -i "s|quay.io/cincinnati/cincinnati-operator:latest|$OPERATOR_IMAGE|" deploy/operator.yaml
+	sed -i "s|quay.io/cincinnati/cincinnati:latest|quay.io/app-sre/cincinnati:2873c6b|" deploy/operator.yaml
```

Comment:
Make this an env variable defaulting to `quay.io/app-sre/cincinnati:2873c6b`?

Reply:
Done but with another way which is more in line with the ReadMe and code.